schema/spacewalk/common/tables/rhnPackageChangeLogData.sql

```diff
@@ -28,8 +28,9 @@ CREATE TABLE rhnPackageChangeLogData
 
 ;
 
-CREATE INDEX rhn_pkg_cld_nt_idx
-    ON rhnPackageChangeLogData (name, time)
+CREATE UNIQUE INDEX CONCURRENTLY rhn_pkg_cld_ntt_idx
```

Comment:
I don't think `CONCURRENTLY` is accepted in a `tables` file because, as far as I know, these scripts are used to create the database from scratch.

Reply:
Good point, thanks.